gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtilsTest.java

```diff
@@ -38,4 +59,299 @@ public void testGetJobSpecFromDag() throws Exception {
       Assert.assertEquals(jobSpec.getConfigAsProperties().get(key), jobSpec.getConfig().getString(key));
     }
   }
+
+  @Test
+  public void testIsDagFinished() throws URISyntaxException {
```

Comment:
given the method lives in `DagProcUtils`, why put it into `DagManagerUtilsTest`?